### PR TITLE
add Simps

### PR DESCRIPTION
**Pull request by sy-records**

### CODE REVIEW

File: frameworks/PHP/simps/app/Model/DbModel.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+namespace App\Model;
+
+use Simps\DB\BaseModel;
+
+class DbModel extends BaseModel
+{
+    public function fortunes()
+    {
+        $fortune = [];
+        $this->pdo->fortune_test = $this->pdo->fortune_test ?? $this->pdo->prepare('SELECT id, message FROM Fortune');
```
Comment (joanhey) [inline frameworks/PHP/simps/app/Model/DbModel.php:21]: For me that don't look like a ful ORM,
You are using the PDO directly and with SQL.

Micro ORM at most.

File: frameworks/PHP/simps/app/Model/DbModel.php [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        $this->pdo->fortune_test->execute();
+        $arr = $this->pdo->fortune_test->fetchAll();
+        foreach ($arr as $row) {
+            $fortune[$row['id']] = $row['message'];
+        }
+        $fortune[0] = 'Additional fortune added at request time.';
+        \asort($fortune);
```
Comment (joanhey) [inline frameworks/PHP/simps/app/Model/DbModel.php:28]: You are sorting an array, not the array of Fortunes objects.

Read the requiriments:
https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#requirements-3

Please update the code, or change the benchmark_config to raw db.

Comment (joanhey) [reply frameworks/PHP/simps/app/Model/DbModel.php:28]: All the db tests will use objects: db, updates and fortunes.

File: frameworks/PHP/simps/app/Model/DbModel.php (hunk 1 above)
Comment (sy-records) [reply frameworks/PHP/simps/app/Model/DbModel.php:21]: I used the pdo connection pool and did not use orm. How should it be modified?

Comment (sy-records) [reply frameworks/PHP/simps/app/Model/DbModel.php:21]: I want to add a separate ORM pr in the future.

File: frameworks/PHP/simps/benchmark_config.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        "fortune_url": "/fortunes",
+        "update_url": "/updates/",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Micro",
+        "database": "mysql",
+        "framework": "Simps",
+        "language": "PHP",
+        "flavor": "None",
+        "orm": "Full",
```
Comment (joanhey) [inline frameworks/PHP/simps/benchmark_config.json:18]:
```suggestion
        "orm": "Raw",
```
If you aren't using the ORM.

Comment (sy-records) [reply frameworks/PHP/simps/benchmark_config.json]: ok.

File: frameworks/PHP/simps/app/Model/DbModel.php (hunk 2 above)
Comment (sy-records) [reply frameworks/PHP/simps/app/Model/DbModel.php:28]: Does this need to be revised again?

Comment (joanhey) [reply frameworks/PHP/simps/app/Model/DbModel.php:28]: No, thanks for the changes.
I liked the code for this fw.

Comment (sy-records) [reply frameworks/PHP/simps/app/Model/DbModel.php:28]: Thanks for the review.

**Comment by sy-records** [comment]
@nbrady-techempower @joanhey Can this pr be merged now?

File: frameworks/PHP/simps/app/Model/DbModel.php [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    public function updates(int $queries = 0)
+    {
+        $query_count = 1;
+        if ($queries > 1) {
+            $query_count = $queries > 500 ? 500 : $queries;
+        }
+
+        $arr = [];
+        $this->pdo->updates_test_select = $this->pdo->updates_test_select ?? $this->pdo->prepare(
+                'SELECT randomNumber FROM World WHERE id = ?'
```
Comment (nbrady-techempower) [inline frameworks/PHP/simps/app/Model/DbModel.php:41]: Need to select `id` here as well.

Other than that, this is ready to go.

**Comment by sy-records** [comment]
@nbrady-techempower I have modified, thank you.

**Pull request merged**